test/e2e/framework/ingress_utils.go

```diff
@@ -62,7 +62,12 @@ const (
 	validFor = 365 * 24 * time.Hour
 
 	// Ingress class annotation defined in ingress repository.
-	ingressClass = "kubernetes.io/ingress.class"
+	// TODO: All these annotations should be reused from
+	// ingress-gce/pkg/annotations instead of duplicating them here.
+	IngressClass = "kubernetes.io/ingress.class"
```

Comment:
Is there any issue with circular vendoring that we have to be careful of?

Reply:
`pkg/annotations` has basic code that does not depend on kubernetes/kubernetes so we should be fine?